src/System.Reflection.DispatchProxy/src/Configurations.props

```diff
@@ -5,6 +5,8 @@
       netfx;
       netstandard;
       netcoreapp2.0;
+      uap10.0.16299-Windows_NT;
+      uap10.0.16299aot-Windows_NT;
       uapaot-Windows_NT;
```

Comment:
Following the netcoreapp2.0 pattern perhaps you should remove the version-less ones into BuildConfigurations.

Reply:
sounds good, I'll do that.